securedrop/session_that_ignores_api.py

```diff
@@ -0,0 +1,17 @@
+from flask import sessions, request
+
+class SessionThatIgnoresAPI(sessions.SecureCookieSessionInterface):
```

Comment:
Happy to take suggestions for a better name here :)

Reply:
I would call this simply `JournalistInterfaceSession` since we may want to add other features in the future. I would also prefer this just be in the module `journalist_app.utils` since I don't think it deserves its own module with a complex name.